_static/my_theme.css

```diff
@@ -136,7 +136,6 @@ h3 {
 
 .nuitka-grid-offer ul li::marker {
 	content: "\f00a";
```

Comment:
I wonder if we broke lists now removing the Font. I guess there is no way other this can be done. The lists are on https://nuitka.net/commercial/resellers.html currently though, but I guess that page then loads the FontAwesome so far still? I guess it's still broken, the page looks ugly now. We need to devise a solution for it. Can these be inline SVG images at all?

Reply:
btw, just removing the font is bound to make it worse, I think without that, it could still render properly due to a local or browser font installation that might exist.